var/spack/repos/builtin/packages/fastjet/package.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Fastjet(AutotoolsPackage):
+    """
+    A high energy physics software package for jet finding in pp
+    and e+e- collisions.
+    """
+
+    homepage = "http://fastjet.fr/"
+    url      = "http://fastjet.fr/repo/fastjet-3.3.3.tar.gz"
+
+    maintainers = ['drbenmorgan', 'vvolkl']
+
+    version('3.3.3', sha256='30b0a0282ce5aeac9e45862314f5966f0be941ce118a83ee4805d39b827d732b')
+    version('3.3.2', sha256='3f59af13bfc54182c6bb0b0a6a8541b409c6fda5d105f17e03c4cce8db9963c2')
+    version('3.3.1', sha256='76bfed9b87e5efdb93bcd0f7779e27427fbe38e05fe908c2a2e80a9ca0876c53')
+    version('3.3.0', sha256='e9da5b9840cbbec6d05c9223f73c97af1d955c166826638e0255706a6b2da70f')
+    version('3.2.2', sha256='3a70cb6ba64071db49a7eecad821679e1a0dadd84e8abca83e518802b3d876e5')
+    version('3.2.1', sha256='c858b6c4f348c3676afa173251bb16d987674e64679a84306510e3963f858d5b')
+    version('3.2.0', sha256='96a927f1a336ad93cff30f07e2dc137a4de8ff7d74d5cd43eb455f42cf5275e3')
+    version('3.1.3', sha256='9809c2a0c89aec30890397d01eda56621e036589b66d7b3cd196cf087c65e40d')
+    version('3.1.2', sha256='dcc834e53da821cbac459c00249d5d18aee6ac866f37551d6a0c60690d3c170b')
+    version('3.1.1', sha256='38303789390726803bd3e7b3a245933273e86342d080b82754df44f5168634eb')
+    version('3.1.0', sha256='f8dc701dfdb124f009b7614010b911e8cc552655c2a966a7f2608a6caa062263')
+    version('3.1.0-beta.1', sha256='8bb6435d7e94ba209b8e39a2bd4cc014bfd89f0adb94a0b39c10eee026c0d7b2')
```

Comment:
You can probably remove all of these alpha and beta releases unless someone needs them. Honestly, I usually only add the latest version of a package.

Reply:
I tend to agree, but  this library is a bit special in that older versions are often needed for validation studies, same goes for HEP generators such as pythia8. But the beta versions are overkill indeed, I removed those.